biomage/stage/stage.py

```diff
@@ -677,9 +677,9 @@ def stage(token, org, deployments, with_rds, auto):
         inputs={
             "manifest": manifest,
             "sandbox-id": sandbox_id,
-            # Convert with_rds to string because Github has issues with boolean inputs
+            # Convert stage_rds to string because Github has issues with boolean inputs
```

Comment:
so now its not a boolean but instead a string or no value?

Reply:
Github workflows can not parse boolean values, so we have to turn this into a string ('False' or 'True') which we then compare in the workflow (e.g. if with-rds == 'True')